documentation/src/main/asciidoc/topics/ref_performance_tuning.adoc

```diff
@@ -79,6 +79,95 @@ The recommended values are:
 * send window size - 640 KB
 * receive window size - 25 MB
 
+== Thread pool settings
+
+A thread pool manages threads by assigning them concurrent tasks.
+Additionally, a thread pool can improve system performance by using any idle thread instead of creating threads.
+
+Thread pools limit the creation and deletion of threads to active worker threads, so that the amount of threads does not typically exceed the amount of tasks.
+This helps reduce the memory footprint of your {brandname} server.
+
+You can configure your `threading` subsystem, so that you can better manage thread pools and their resources. 
```

Comment:
I would say we should put a NOTE or something saying that normally this should not be done and unless a user detects and actual reason to change this they should not.

Reply:
Updated to include a note:

> Consider only configuring the `threading` subsystem when necessary, such as configuring it to define a behavior that meet your needs.